src/constants/rdf_predicates.js

```diff
@@ -0,0 +1,8 @@
+import { SCHEMA_INRUPT } from '@inrupt/vocab-common-rdf';
+
+const RDF_PREDICATES = {
+    ...SCHEMA_INRUPT,
+    uploadDate: 'https://schema.org/uploadDate'
+}
+
+export default RDF_PREDICATES;
```

Comment:
Same as src/constants/index.jsx. I we can add a blank line at the end of the file to keep things consistent with the rest of our files.